Release - v1.4.0

uguraslan opened the pull request:
## Test Package

~[lightning-blits-1.4.0.vsix.zip](https://github.com/user-attachments/files/19079484/lightning-blits-1.4.0.vsix.zip)~

[lightning-blits-1.4.0.vsix.zip](https://github.com/user-attachments/files/19161380/lightning-blits-1.4.0.vsix.zip)


## Changes

- Fixes syntax highlighting issue for `@updated` event attribute
- Auto file template provider for newly created `.blits` files
- Snippet support for JavaScript/TypeScript/Blits files
- The bundled extension file size reduced

### Snippets

https://github.com/user-attachments/assets/bbbb3942-eabe-4b1e-9659-9cba24dc7296

### Auto `.blits` file templates

https://github.com/user-attachments/assets/b8fbab88-4bfe-4806-a0bc-4f51853a6ddd

suresh-gangumalla commented:
Blits snippet prefixes are not listing except `blits-file` prefix in .blits file template section. 
atleast `blits-attribute-transition` snippet prefix should be listed. 

![Blits-snippet-prefix-suggestions-not-listing](https://github.com/user-attachments/assets/52adcf21-5921-4b59-9863-8204a0fc8f2f)

uguraslan commented:
Hey @suresh-gangumalla, in JavaScript/TypeScript scope (inside script tags) in `.blits` files all snippets are available. So the suggested list changes for `.blits` files based on where the cursor is.

But `blits-attribute-transition` was missing in the template section of `.blits` files, I have just added it and updated the test package.

suresh-gangumalla reviewed (approved):
LGTM
